### ISSUE TITLE
make our containers citable

### ISSUE THREAD

**Issue by stebo85** [NeuroDesk/neurocontainers#142]
Should we upload all our singularity containers that are released in apps.json to zenodo and assign them a DOI so people can cite them nicely? 

Could be integrated in the script that currently uploads the containers to object storage?

**Comment by stebo85** [comment]
upload limit to zenodo per item is 50gb

**Comment by thomshaw92** [comment]
OSF? :) 
UQ eSpace?

**Comment by xchen101** [comment]
Hi, I see that the NeuroDesk project is funded by ARDC, which is a consortium member of DataCite, they should be able to help you with registering DOIs for the containers to make them citable. This would make the applications significantly FAIRer :)

**Comment by stebo85** [comment]
Yes, the problem is currently not the doi itself but finding long term storage for the containers and automating the upload and doi generation. We are currently planning to upload our containers to Zenodo. Once we completed our cloud provider move from oracle to aws we will start working on this issue again

**Comment by iishiishii** [comment]
Add a step to upload to zenodo in container builder action

**Comment by stebo85** [comment]
is this completed?
